src/Microsoft.AspNetCore.Blazor.BuildTools/Core/ILWipe/MethodWipedExceptionMethod.cs

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+using System;
+
+namespace Microsoft.AspNetCore.Blazor.BuildTools.Core.ILWipe
+{
+    static class MethodWipedExceptionMethod
+    {
+        public static MethodDefinition AddToAssembly(ModuleDefinition moduleDefinition)
+        {
+            // Adds the following method to the assembly:
+            // namespace ILWipe
+            // {
+            //     internal static class ILWipeHelpers
+            //     {
+            //         public static Exception CreateMethodWipedException()
+            //         {
+            //             return new NotImplementedException("Cannot call method because it was wiped. See stack trace for details.");
+            //         }
```

Comment:
Encapsulating this logic in a shared method means that all the places where it's used (the "wiped" methods) involve fewer repeated IL instructions. Plus it saves us the trouble of declaring a string const to avoid repeating that.